react/features/base/media/components/web/AudioTrack.js

```diff
@@ -244,7 +243,8 @@ export default class AudioTrack extends Component<Props> {
                 } else {
                     this._playTimeout = undefined;
                 }
-            });
+            }) // catch any errors that can potentially occur in the then clause.
+            .catch(e => logger.error(e));
```

Comment:
what's the point in catching those errors exactly? I'd think it's okay for them to be unhandled, so they can be reported by whatever mechanisms detects unhandled errors (global error handler?)

Reply:
We call this method in componentDidMount and also shouldComponentUpdate. I would like to avoid having unhandled promises, exceptions, errors, etc. Also the global error handler will do exactly `logger.error`.

Reply:
You could use the global error handler for monitoring unhandled errors across the app and sending them somewhere. This kind of errors that are not expected and should be fixed rather than handled.

Reply:
I was more worried about the `sendAnalytics` call. It can fail any reason (no network connection, some analytics server down, etc). But now I see that we have `try {...} catch` there. I'm fine to remove the catch here.

Reply:
I see, yeah, it wouldn't make sense to bubble up network failures to the global handler.

Reply:
removed the `catch`